### PR TITLE
Fixes PHPCS Errors based on WPCS

### LINKED ISSUE #33: Enforce WordPress Coding Standards on the repo
We should clean up the codebase and then enforce WPCS.

### PR DESCRIPTION
**Pull request by kopepasah**
This covers a first stab at fixing some of the PHPCS errors, which will enable us to test next iterations of this plugin more easily. 

Currently there is one final error to fix and it is blocked by #48.

Fixes #33

### CODE REVIEW

**Comment by kopepasah** [comment]
Please do note that, at the creation of this PR, I have not tested these changes functionally. That is the next step.

File: pantheon-hud.php [hunk 1]
```diff
@@ -8,35 +8,38 @@
...
-	$file = dirname( __FILE__ ) . '/inc/' . str_replace( '_', '-', strtolower( implode( $parts, '/' ) ) );
-	if ( file_exists( $file ) ) {
-		require $file;
+		$parts = explode( '\\', $class );
+		array_shift( $parts ); // Don't need "Pantheon".
+		array_shift( $parts ); // Don't need "HUD".
+		$last    = array_pop( $parts ); // File should be 'class-[...].php'.
+		$last    = 'class-' . $last . '.php';
+		$parts[] = $last;
+		$file    = dirname( __FILE__ ) . '/inc/' . str_replace( '_', '-', strtolower( implode( $parts, '/' ) ) );
```
Comment (danielbachhuber) [inline pantheon-hud.php]:
```suggestion
		$file    = dirname( __FILE__ ) . '/inc/' . str_replace( '_', '-', strtolower( implode( '/', $parts ) ) );
```

File: phpcs.xml.dist [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+<ruleset name="WordPress Coding Standards">
+	<config name="installed_paths" value="vendor/wp-coding-standards/wpcs" />
 
+	<rule ref="WordPress" />
+
+	<arg name="extensions" value="php" />
+
+	<file>.</file>
+	<exclude-pattern>/vendor/</exclude-pattern>
 </ruleset>
```
Comment (danielbachhuber) [inline phpcs.xml.dist]: Can we use a derivative of https://github.com/pantheon-systems/wp-saml-auth/blob/master/phpcs.xml.dist ? It has a few more dodads that are useful.

Comment (kopepasah) [reply phpcs.xml.dist]: Yeah, sound good. I can do this tomorrow (Tuesday).

**Comment by kopepasah** [comment]
@danielbachhuber I updated the PHPCS config with the config file you reference and removed the unnecessary configuration. 

I did not change the PHP or WP version compatibility requirement configuration, as I am not of those without further digging. Hoping @ataylorme or yourself will know those minimum requirements. Either way, these need to be updated.

Let me know if there is something else missing.

File: pantheon-hud.php [hunk 3]
```diff
@@ -1,42 +1,45 @@
 <?php
 /**
  * Plugin Name: Pantheon HUD
- * Version: 0.2.2
+ * Version: 0.2.1
```
Comment (danielbachhuber) [inline pantheon-hud.php]:
```suggestion
 * Version: 0.2.2
```

**Comment by danielbachhuber** [comment]
Looks great, thanks for your work on this!

**Pull request merged**